test/quantization/dbr/test_quantize_dbr.py

```diff
@@ -418,6 +419,8 @@ def forward(self, x):
         qconfig = torch.quantization.default_qconfig
         self._test_auto_tracing(m, qconfig, (torch.randn(1, 1, 2, 2),))
 
+    @unittest.skip("This works in DBR, but doesn't work anymore in fx graph mode after "
```

Comment:
cc @vkuzo can we disable this for now? after lowering refactor we don't have a good way to support repeated use of the same module in fx graph mode quant right now, but I think it's probably not a very important case, we can brainstorm ways to support this if needed in the future, or we can just detect it and error out (I remember you have a PR that detects somethings similar)

Reply:
This is an edge case, we do have at least team in Meta who use it in their models but it is not a common thing.  I think it's reasonable to remove it from FX graph mode quantization for now and add it back later if it's needed.

Instead of skipping this test, we could set `do_fx_comparison` to `False` in the `self._test_auto_tracing` function call, this is also used for features which are supported in DBR but not FX graph mode.